leaflet/utils.py

```diff
@@ -68,13 +44,6 @@ def __add__(self, iterable):
         lazy_list.extend(iterable)
         return lazy_list
 
-    @classmethod
-    def _resolve_lazy_item(cls, item):
```

Comment:
Why did you remove that?

Reply:
It wasn't needed anywhere else as far as I can tell, so I just dropped it. Can revert that ofc.

Reply:
It is used in `__iter__` / `__getitem__` just above. And as far as I can tell it as nothing to do with the memoize. I'd rather let `ListWithLazyItems` as it was before to avoid any bug.

Reply:
We can't use the default Django `lazy` function then because the `__proxy__` class doesn't define a `__call__` method which is used in the old `_resolve_lazy_items`.

Reply:
We actually don't even need the custom lazy stuff in `utils.py` if we use the standard Django `lazy` function, so I just reverted those changes and modified `__init__.py` accordingly.